### DISCUSSION TITLE
Plans for Vite 6?

### DISCUSSION THREAD

**Discussion by hubdidub** [quasarframework/quasar#17627] [General - Components / Directives / etc] [answered]
Are there any plans for supporting Vite v6 yet?

Looks like the new q/app-vite major version is getting ready to be fully released, in which we bump Vite to v5.
Since Vite 6 is already in beta, I'm wondering if it might be worth waiting for it and putting it into the q/app-vite release?

I have no idea how big the jump from v5 to v6 is and if it will be anywhere near as difficult as the previous versions.
Do you think it will require another major version of q/app-vite?

**Comment by rstoenescu** [comment, marked as answer]
The very next q/app-vite v2 beta.25 will come with Vite 6

**Reply by hubdidub** [reply]
nice :)
